devkit/loadtesting-k6/README.md

```diff
@@ -5,30 +5,71 @@
 This package utilizes k6 by Grafana Labs.
 Install k6 on test machine: https://k6.io/docs/get-started/installation/
 
-## How to test RPC Server
+## Build
+1. Navigate to `devkit/loadtesting-k6`
+2. Build the repo
+```
+yarn build
+```
 
-1. Spin up Sandbox
-2. Spin up RPC Server on http://localhost:8080/
-3. Navigate to `.\devkit\k6-loadtesting\`
-4. Run:
-   - `yarn start:artificial` (Artificial usage with `eth_getCode` and `eth_call`)
-   - `yarn start:blockwallet` (Real usage of blockwallet)
-   - `yarn start:trustwallet` (Real usage of trustwallet)
-   - `yarn start:burst` (Run various burst tests)
-   - `yarn start:constant` (Run various load tests for 10 seconds each)
-   - `yarn start:long` (Run long lived load tests for 1 minute each)
+## Test
+### How to test RPC Server
+
+1. Spin up Sandbox or all the infrastracture locally (except for the entry/exit nodes) or spin up a RPCh node of staging environment where the RPC Server runs on http://localhost:45740/. Please refer to [DEVELOPER_SETUP](../../DEVELOPER_SETUP.md) for more details
+2. Navigate to `devkit/loadtesting-k6`
+3. Run:
+```bash
+export TEST_TYPE=SMOKE WALLET_TYPE=METAMASK ; \
+   RPC_SERVER_URL=http://localhost:45750 \
+   k6 run ./dist/rpc-server/index.js \
```

Comment:
should be `yarn exec` to avoid calling global binaries

Reply:
Just noticed that `"k6"` shouldn't appear in the `package.json`. For this command, it's expected to call the (global) installed binary of "k6"

Reply:
Any reason why this should be a global dependency?

Reply:
Given `yarn exec` restricts the execution environment to current workspace, the assumption that it's in the current workspace that users have installed "k6"  should hold for the command to be executable. As we do not provide script for installing k6 under the project's folder, the command may fail when it's wrapped with `yarn exec`.

Reply:
It's a dependency in the `package.json`, isn't it?

Reply:
As per DM, it's resolved